docs/source/references/schema.json

```diff
@@ -602,14 +602,14 @@
         "name": "header",
         "type": "BlockHeader"
       },
-      {
```

Comment:
Wasn't this supposed to be stable?

Reply:
it's the order of fields in the struct you are referring to? Should I revert it?